Stripe/STPPaymentContext.m

```diff
@@ -145,6 +146,7 @@ - (void)retryLoading {
                 [self.loadingPromise fail:error];
                 return;
             }
+            self.shippingAddress = customer.shipping;
```

Comment:
I think we need to add something in here to distinguish between addresses that are auto-filled in from the server and addresses which have been definitively viewed and approved by the user. I think also maybe payment context should not skip the "show shipping VC" step of request payment if the user has not verified this autofilled address (or maybe that behavior should be configurable).